### PR TITLE
update 1.13

### PR DESCRIPTION
**Pull request by martabereta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Red Hat OpenShift Container Platform 4.2 or newer installed on Linux x86_64, Lin
 
 ## Operator versions
 
-- 1.0.0, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.2.2, 1.2.3, 1.3.1, 1.4.1, 1.5.0, 1.6.0, 1.7.0, 1.8.0, 1.9.0, 1.10.0, 1.11.0, 1.12.0
+- 1.0.0, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.2.2, 1.2.3, 1.3.1, 1.4.1, 1.5.0, 1.6.0, 1.7.0, 1.8.0, 1.9.0, 1.10.0, 1.11.0, 1.13.0
```
Comment (kacperkrzyzak) [inline README.md:24]: Please add here 1.13 instead of replacing 1.12

File: bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -71,11 +71,11 @@ metadata:
...
     categories: Monitoring
     certified: "false"
-    containerImage: quay.io/opencloudio/ibm-licensing-operator:1.12.0
+    containerImage: quay.io/opencloudio/ibm-licensing-operator:1.13.0
     createdAt: "2020-06-22T10:22:31Z"
     description: The IBM Licensing Operator provides a Kubernetes CRD-Based API to monitor the licensing usage of products.
-    olm.skipRange: '>=1.0.0 <1.12.0'
-    operators.operatorframework.io/builder: operator-sdk-v1.11.0
+    olm.skipRange: '>=1.0.0 <1.13.0'
+    operators.operatorframework.io/builder: operator-sdk-v1.12.0
```
Comment (kacperkrzyzak) [inline bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml:78]: Let's leave builder annotation as v1.11.0

**Review by kacperkrzyzak — CHANGES_REQUESTED** [review]

File: bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (SzymonKowalczyk) [reply bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml:78]: we do not update `operator-sdk-v1`

**Review by kacperkrzyzak — APPROVED** [review]

**Pull request merged**